scripts/comsos_multinode.sh

```diff
@@ -0,0 +1,274 @@
+#/bin/sh
+
+display_usage() {
+    printf "** Please check the exported values:: **\n Daemon : $DAEMON\n Denom : $DENOM\n ChainID : $CHAINID\n DaemonHome : $DAEMON_HOME\n \n Github URL : $GH_URL\n Chain Version : $CHAIN_VERSION\n"
+    exit 1
+}
+
+if [ -z $DAEMON ] || [ -z $DENOM ] || [ -z $CHAINID ] || [ -z $DAEMON_HOME ] || [ -z $GH_URL ] || [ -z $CHAIN_VERSION ]
+then 
+    display_usage
+fi
+
+# read no.of nodes to be setup
+NODES=$1
+if [ -z $NODES ]
+then
+    NODES=2
+fi
+
+# read no of accounts to be setup
+ACCOUNTS=$2
+
+echo " ** Number of nodes : $NODES and accounts : $ACCOUNTS has been passed **"
+
+echo "**** Number of nodes to be setup: $NODES ****"
+
+command_exists () {
+    type "$1" &> /dev/null ;
+}
+
+cd $HOME
+
+
+if command_exists go ; then
+    echo "Golang is already installed"
+else
+  echo "Install dependencies"
+  sudo apt update
+  sudo apt-get -y upgrade
+  sudo apt install build-essential jq -y
+
+  wget https://dl.google.com/go/go1.16.8.linux-amd64.tar.gz
+  tar -xvf go1.16.8.linux-amd64.tar.gz
+  sudo mv go /usr/local
+  rm go1.16.8.linux-amd64.tar.gz
+
+  echo "------ Update bashrc ---------------"
+  export GOPATH=$HOME/go
+  export GOROOT=/usr/local/go
+  export GOBIN=$GOPATH/bin
+  export PATH=$PATH:/usr/local/go/bin:$GOBIN
+  echo "" >> ~/.bashrc
+  echo 'export GOPATH=$HOME/go' >> ~/.bashrc
+  echo 'export GOROOT=/usr/local/go' >> ~/.bashrc
+  echo 'export GOBIN=$GOPATH/bin' >> ~/.bashrc
+  echo 'export PATH=$PATH:/usr/local/go/bin:$GOBIN' >> ~/.bashrc
+
+  source ~/.bashrc
+
+  mkdir -p "$GOBIN"
+  mkdir -p $GOPATH/src/github.com
+
+  go version
+fi
+
+echo "--------- Install $DAEMON ---------"
+git clone $GH_URL && cd $(basename $_ .git)
+git fetch && git checkout $CHAIN_VERSION
+make install
+
+cd $HOME
+
+# check version
+$DAEMON version --long
+
+# export daemon home paths
+for (( a=1; a<=$NODES; a++ ))
+do
+    export DAEMON_HOME_$a=$DAEMON_HOME-$a
+    echo "Deamon path :: $DAEMON_HOME-$a"
+
+    $DAEMON unsafe-reset-all  --home $DAEMON_HOME-$a
+    echo "****** here command $DAEMON unsafe-reset-all  --home $DAEMON_HOME-$a ******"
+done
+
+rm -rf ~/.$DAEMON/config/gen*
```

Comment:
This line will only delete `~/.regen/config/gen`. It would be better to remove all the `$DAEMON_HOME-$a` directories

Reply:
okay.